shared/engine/index-impl.tsx

```diff
@@ -327,21 +323,21 @@ class FakeEngine {
       sessionID: 0,
     })
   }
-  _channelMapRpcHelper(configKeys: Array<string>, method: string, params: any) {
+  _channelMapRpcHelper(_: Array<string>, __: string, ___: any) {
     return null
   }
   _rpcOutgoing(
-    method: string,
-    params: {
+    _: string,
```

Comment:
Why not just prefex their original names with an `_`?

Reply:
Just less 'stuff' to visually parse.  If it's not handling it then it likely doesn't need anything more than that. Could be convinced otherwise

Reply:
I didn't realize this was inside fakeEngine, nvm